FtpClient/FtpCli/Pkgs/SftpClient.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using Renci.SshNet;
+using Renci.SshNet.Sftp;
+
+namespace FtpCli.Packages.ClientWrapper
+{
+
+    public class Client : IClient
+    {
+        private SftpClient _client;
+        private ConnectionInfo _connection;
+        private bool _isConnected;
+        private delegate void UseCallback(SftpClient client);
+
+        // Explicit constructor
+        // Should be used in all cases
+        public Client(string host, int port, string username, string password)
+        {
+             _connection = new ConnectionInfo(host, port, username,
+                 new PasswordAuthenticationMethod(username, password)); 
+
+             // All operations will be performed through the '_client'
+             _client = new SftpClient(_connection);
+
+             try {
+                _client.Connect();
+                _isConnected = true;
+             } catch (Exception err) 
+             {
+                Console.WriteLine($"Unable to connect to {host}:{port}");
+                Console.WriteLine(err.ToString());
+                Environment.Exit(-1);
+             }
+        }
+
+
+        public bool IsConnected {
+          get { return _isConnected; }
```

Comment:
Is there any case where this returns false?

Reply:
It can. When it is time to implement timing out from the server, this can be used in the main event loop to exit upon disconnecting. I realize I should change this to return the library

The library (SSH.NET) Connect() method can be invoked with a Timeout, which will be useful.